SeaPublicWebsite/Views/Cookie/CookieSettings.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@model SeaPublicWebsite.Models.Cookies.CookieSettingsViewModel
+@{
+    ViewBag.Title = $"Cookies on GOV.UK";
+}
+
+<div class="govuk-grid-row cookie-settings-page">
+    <div class="govuk-grid-column-two-thirds">
+        @if (Model.ChangesHaveBeenSaved)
+        {
+            <div class="govuk-notification-banner govuk-notification-banner--success" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
+              <div class="govuk-notification-banner__header">
+                <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                  Success
+                </h2>
+              </div>
+              <div class="govuk-notification-banner__content">
+                <p class="govuk-notification-banner__heading">
+                  You’ve set your cookie preferences.
+                </p>
+              </div>
+            </div>
+        }
+
+        <form method="POST" action="@Url.Action("CookieSettings_Post", "Cookie")">
+            @(Html.AntiForgeryToken())
+            <h1 class="govuk-heading-xl">Cookies on GOV.UK</h1>
```

Comment:
A more general question that may need to be asked to BEIS technical people.
This message makes it sound like we're setting cookies for all of Gov.UK when we're actually only doing it for our service. If all services follow this pattern then users of various Gov.UK services will keep seeing the same cookie banner and may wonder why they keep getting asked the same question.
Is that correct, or are we missing something?

Reply:
We can use 'our service', to avoid confusion and to keep it short.